### DISCUSSION TITLE
Display table items as marketplace catalog

### DISCUSSION THREAD

**Discussion by hypknotics** [Budibase/budibase#2777] [Help]
Hi, I am using budibase to build my e-commerce app. Is there a way to display budibase database table items as a marketplace catalog. Ideally I want to define a catalog item template and then hydrate it with each row in the table to create a marketplace. Thanks.

**Comment by mjashanks** [comment]
Hey @hypknotics 

You could try using a Data Provider + Repeater: https://docs.budibase.com/building-apps/design/components/repeater

Word or warning - an issue that you may encounter if building a market place, is that budibase does not have a concept of self signup. Your users cannot come along and create their own account. User accounts can only be created by existing users. This is by design, as Budibase is built for internal tools.
